src/game_inventory.cpp

```diff
@@ -534,6 +534,19 @@ class comestible_inventory_preset : public inventory_selector_preset
                 return string_format( _( "%.2f%s" ), converted_volume, volume_units_abbr() );
             }, _( "VOLUME" ) );
 
+            append_cell( [&p](const item_location & loc) {
+                const item &it = *loc;
+                const int charges = std::max( it.charges, 1 );
+                const double converted_weight = convert_weight ( it.weight() / charges );
+                // Prevent div by 0
+                if ( converted_weight == 0 ) {
+                    return std::string( "---" );
+                }
+                const nutrients nutr = p.compute_effective_nutrients( *loc );
+                const double calpergr = round_up( nutr.kcal / converted_weight, 2 );
+                return string_format(_( "%.2f" ), calpergr );
+            }, _( "CAL/G" ) );
```

Comment:
Grams should at least be lower case, though potentially the whole thing should.

Reply:
I'll change it to lower case.

Reply:
All other table headings are in all-caps, so it looks a bit strange to have this one in lowercase.

Reply:
That was my original thought, however having cal and g in lowercase better signifies that they are units of measurement and not abbreviations. CAL/g would work too.